# RequestUndock.srv
---
bool success
string message